src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -2171,12 +2171,8 @@ static int client_receive_message_udp(
                 return 0;
         }
 
-        struct timeval *tv = CMSG_FIND_AND_COPY_DATA(&msg, SOL_SOCKET, SCM_TIMESTAMP, struct timeval);
-        if (tv)
-                triple_timestamp_from_realtime(&t, timeval_load(tv));
-
         log_dhcp_client(client, "Received message from UDP socket, processing.");
-        r = client_handle_message(client, message, len, &t);
+        r = client_handle_message(client, message, len, TRIPLE_TIMESTAMP_FROM_CMSG(&msg));
```

Comment:
Beforehand this wasn't initialized if not found, but now it will be initialised at triple_timestamp_now(), is that ok? Same below

Reply:
Yes, that's intended. Anyway, the timestamp will be initialized if not by cmsgbin e.g. dhcp_lease_set_timestamp().